styles.css

```diff
@@ -0,0 +1,523 @@
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:ital,wght@0,300;0,400;0,700;0,900;1,300;1,400;1,700;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+
+/*////////////////////////
+    Table of Contents
+    1. Global Styles
+    2. Typography
+    3. Layout
+        3.1 Reusables
+        3.2 Other
+    4. Areas
+        4.1 Header
+        4.2 Introduction
+        etc.
+    5. Media Queries
+////////////////////////*/
+
+/*Paars #5722CD;*/
+/*Blauw #4845E4;*/
+/*Roze #ED4385;*/
+/*Zwart #0C0C0C (voor standaard tekst);*/
+/*Lichtpaars #F2F0FF (voor de grote zwevende woorden, zoals "hello" en "services") Uitzonderingen:*/
+/*                                                                                 Lichtgrijs #F3F6F9 (voor de achtergrond van de projecten)*/
+/*Grijs #7A7A7A (voor de service-tegel tekst);*/
+
+
+/* ------------------------------- 1. Global Styles -------------------------------- */
+:root{
+    --paars: #5722CD;
+    --blauw: #4845E4;
+    --lichtpaars: #F2F0FF;
+    --lichtgrijs: #F3F6F9;
+    --roze: #ED4385;
+    --zwart: #0C0C0C;
+    --grijs: #7A7A7A;
+
+}
+*{  /*reset css*/
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+body{
+    overflow-x: hidden;
+}
+
+/* ------------------------------- 2. Typography ----------------------------------- */
+*{
+    font-family: roboto,sans-serif;
+    font-style: normal;
+}
+h1,h2,h3,h4,h5,h6{
+    font-family: 'Merriweather', serif;
+}
+h1{
+    font-size:2.4rem;
+}
+.small-weight-p{
+    font-weight: 300;
+}
+a{
+    text-decoration: none;
+    color: var(--paars);
+}
+a:hover{
+    color: rgba(87, 34, 205, 0.7);
+}
+
+/* ------------------------------- 3. Layout --------------------------------------- */
+/* Header {Nav, Hero}*/
+header {
+    width: 101%;
+    height: 560px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+nav{
+    width: 100%;
+    height: 70px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: flex-end;
+    margin-right: 80px;
```

Comment:
Je kunt beter zorgen dat de `ul` die in deze navigatiebalk staat een flex-basis (als maximale breedte) krijgt, zodat 'ie altijd op dezelfde afstand van de rand staat als de andere content. Door zoveel horizontale margin te gebruiken werk je een beetje om flexbox heen